board/safety/safety_gm.h

```diff
@@ -16,6 +16,8 @@ const int GM_MAX_RATE_DOWN = 17;
 const int GM_DRIVER_TORQUE_ALLOWANCE = 50;
 const int GM_DRIVER_TORQUE_FACTOR = 4;
 
+const int GM_STANDSTILL_THRSLD = 11;  // 0.3421kph
+
```

Comment:
All wheel speeds are 0.0311 (10 in raw units) as the highest speed you can engage with your foot on the brake, however safety tests work with `<` not `<=`, so use 11

Reply:
And on other cars (like Silverado, the minimum wheel speed is around 0.6 kph, so the standstill for them is not affected